wg-byon-build-pipelines/README.md

```diff
@@ -16,6 +16,7 @@ The [charter](charter.md) defines the scope and governance of the BYON build pip
 
 ## Meetings
 * Regular WG Meeting: [Wednesdays at 10:00 ET (East Time)](https://meet.google.com/qkt-yacp-wzm) (weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=10:00&tz=ET%20%28East%20Time%29).
+    * [Meeting Notes and Agenda](https://docs.google.com/document/d/1i1ET2HHqWry-0NK72iDRCEGEoyO7CPzmwOBjr437CVI/edit?usp=sharing)
```

Comment:
This change will get overwriten by any change to `sigs.yaml`. To add a line like this edit `sigs.yaml` here
https://github.com/open-services-group/community/blob/b05d0c14d1b63707d7e56e92bbb1d09eea2ccd44/sigs.yaml#L165-L170

And add a `archive_url` property (example here: https://github.com/open-services-group/community/blob/b05d0c14d1b63707d7e56e92bbb1d09eea2ccd44/sigs.yaml#L76 )

Then execute
https://github.com/open-services-group/community/blob/b05d0c14d1b63707d7e56e92bbb1d09eea2ccd44/Containerfile#L16

This command will regenerate the README.md based on the `sigs.yaml` prescription

Reply:
Thanks @tumido Made the changes :)